configure.ac

```diff
@@ -1860,8 +1903,15 @@ dnl Find the X11 include and library directories
 CheckX11()
 {
     AC_ARG_ENABLE(video-x11,
-[AS_HELP_STRING([--enable-video-x11], [use X11 video driver [default=yes]])],
-                  , enable_video_x11=yes)
+[AS_HELP_STRING([--enable-video-x11], [use X11 video driver [default=maybe]])],
+                  ,[
+    enable_video_x11=yes
+    case "$host" in
+        *-*-darwin*|*-ios-*)
+            enable_video_x11=no
+            ;;
+    esac])
+
```

Comment:
I've got a question left about this.
This disables x by default on Macos, and keeps it enabled on other platforms.

Perhaps you prefer to keep it always enabled, and only disable it on CI?

Reply:
No, it should default to always disabled on macOS.

Reply:
Then I'll merge it asap.
I just rebased it to make sure no new warnings were introduced in the last few days.